docs/0.1/grid_product_catalog_family_specification.md

```diff
@@ -0,0 +1,496 @@
+# Grid Product Catalog Specification
+
+<!--
+  Copyright (c) 2020 Cargill Incorporated
+  Copyright (c) 2019 Target Brands, Inc.
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+## Overview
+
+Grid Product Catalog is a smart contract designed to run with the
+[Sawtooth Sabre](https://github.com/hyperledger/sawtooth-sabre/)
+smart contract engine.
+
+Grid Product Catalog is designed for sharing an assortment of products between
+organizations. Each product in a catalog will contain a number of master data
+elements. Using Grid Product Catalog, organizations can share collections of
+products with other participants in their network.
+
+This specification describes the available data objects, state addressing (how
+transaction information is stored and addressed by *namespace*), and the valid
+transactions: types, headers, payload format, and execution rules.
+
+## State
+
+All Grid Product Catalog objects are serialized using protocol buffers
+(protobufs) before being stored in state. These objects include Catalogs and
+Products.
+
+### Catalog
+
+Catalogs are the primary entities stored in state for the Grid Product Catalog
+smart contract. It has the following fields:
+
+* catalog_id: The unique identifier for a catalog.
+* owner: The ID of the Pike organization that created the catalog.
+* name: A human readable name for the catalog.
+* properties: A list of property values (as defined in Grid Schema) which define
```

Comment:
Per the new naming conventions, it should be "the Schema smart contract" (without Grid, and with "smart contract" because it's the first mention of Schema).